diff --git a/node_modules/expo-live-activity/plugin/build/lib/getWidgetFiles.js b/node_modules/expo-live-activity/plugin/build/lib/getWidgetFiles.js
index 40cf3b3..e9b9728 100644
--- a/node_modules/expo-live-activity/plugin/build/lib/getWidgetFiles.js
+++ b/node_modules/expo-live-activity/plugin/build/lib/getWidgetFiles.js
@@ -1,22 +1,22 @@
 "use strict";
-var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     var desc = Object.getOwnPropertyDescriptor(m, k);
     if (!desc || ("get" in desc ? !m.__esModule : desc.writable || desc.configurable)) {
-      desc = { enumerable: true, get: function() { return m[k]; } };
+        desc = { enumerable: true, get: function () { return m[k]; } };
     }
     Object.defineProperty(o, k2, desc);
-}) : (function(o, m, k, k2) {
+}) : (function (o, m, k, k2) {
     if (k2 === undefined) k2 = k;
     o[k2] = m[k];
 }));
-var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function(o, v) {
+var __setModuleDefault = (this && this.__setModuleDefault) || (Object.create ? (function (o, v) {
     Object.defineProperty(o, "default", { enumerable: true, value: v });
-}) : function(o, v) {
+}) : function (o, v) {
     o["default"] = v;
 });
 var __importStar = (this && this.__importStar) || (function () {
-    var ownKeys = function(o) {
+    var ownKeys = function (o) {
         ownKeys = Object.getOwnPropertyNames || function (o) {
             var ar = [];
             for (var k in o) if (Object.prototype.hasOwnProperty.call(o, k)) ar[ar.length] = k;
@@ -45,7 +45,9 @@ function getWidgetFiles(targetPath) {
     catch {
         console.log('Building for example app');
     }
-    const liveActivityFilesPath = path.join(packagePath ? packagePath : '..', '/ios-files');
+    const liveActivityFilesPath = fs.existsSync(path.join(process.cwd(), 'widgets'))
+        ? path.join(process.cwd(), 'widgets')
+        : path.join(packagePath ? packagePath : '..', '/ios-files');
     const imageAssetsPath = './assets/liveActivity';
     const widgetFiles = {
         swiftFiles: [],
